content/extraProblems.json

```diff
@@ -589,6 +589,149 @@
         "usacoId": "1166"
       }
     },
+    {
+      "uniqueId": "usaco-1179",
+      "name": "Herdle",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1179",
+      "source": "Bronze",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["Simulation"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1179"
+      }
+    },
+    {
+      "uniqueId": "usaco-1180",
+      "name": "Non-Transitive Dice",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1180",
+      "source": "Bronze",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Brute Force"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1180"
+      }
+    },
+    {
+      "uniqueId": "usaco-1181",
+      "name": "Drought",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1181",
+      "source": "Bronze",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Greedy", "Math"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1181"
+      }
+    },
+    {
+      "uniqueId": "usaco-1183",
+      "name": "Cow Frisbee",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1183",
+      "source": "Silver",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Sorted Set", "Stack", "Linked List"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1183"
+      }
+    },
+    {
+      "uniqueId": "usaco-1184",
+      "name": "Cereal 2",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1184",
+      "source": "Silver",
+      "difficulty": "Very Hard",
+      "isStarred": false,
+      "tags": ["Spanning Tree", "Constructive"],
```

Comment:
I know this is late - but connected components? Did not see an official solution posted yet so idk what your sol. was.

Reply:
check the analysis. of course, you'll need to find the connected components before finding a spanning tree for each component